dotnet-desktop-guide/net/wpf/events/snippets/how-to-create-a-custom-routed-event/vb/MainWindow.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<!--<MainWindowXaml>-->
+<Window x:Class="CodeSampleVb.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:custom="clr-namespace:WpfControlVb;assembly=WpfControlLibraryVb"
```

Comment:
If you get this fixed before me, no worries 😁 the namespace has to change to the one defined in the C# library
```suggestion
        xmlns:custom="clr-namespace:WpfControl;assembly=WpfControlLibrary"
```